scripts/release/create_bin_release.sh

```diff
@@ -44,7 +44,7 @@ PRODUCT_NAME=${PRODUCT_NAME}-${VERSION}
 rm -rf ${PRODUCT_NAME}
 mkdir ${PRODUCT_NAME}
 
-git clone https://github.com/mrproliu/skywalking-rover.git ./${PRODUCT_NAME}
+git clone https://github.com/apache/skywalking-rover.git ./${PRODUCT_NAME}
```

Comment:
When is this mis-submitted? Is this affected the last vote release?

Reply:
It's won't affect the latest release. I have change this code before each release.